torch/csrc/jit/interned_strings.h

```diff
@@ -64,7 +64,7 @@ _(perm) \
 _(shape) \
 _(axes) \
 _(group) \
-_(__inplace)
+_(inplace)
```

Comment:
So, how are we preventing someone from defining an ATen parameter that clobbers this attribute?

Reply:
ATen functions manage their attributes in their own way. Only PythonOps get it assigned automatically, but they have no other attributes anyway. Having a consistent name for all kinds of ops help when doing the checks

Reply:
Yes, my point is that if someone adds a parameter to an ATen function called "inplace", the automatic recordTrace translation will clobber the "inplace" annotation on the node.